modules/ROOT/pages/changelog.adoc

```diff
@@ -4,6 +4,78 @@
 
 NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].
 
+== 6.5 - 2023-06-01
+
+=== Added
+* New `table_merge_content_on_paste` option which disables the merging behaviour when pasting a table inside an existing table.
+* New optional `defaultExpandedIds` and `onToggleExpand` options to the `tree` component config.
+* New optional `defaultSelectedId` option to the `tree` component config.
+* New `help_accessibility` option which displays the keyboard shortcut to access the help functionality in the statusbar.
+* New `accordion` plugin with the `InsertAccordion` command.
+* New `accordion` and `accordion-toggle` icons.
+* New `details_initial_state` and `details_serialized_state` options.
+* New `init_content_sync` option that initializes the editor iframe using `document.write` instead of `srcdoc`.
+* New `newdocument_content` option that changes the content that will be set when using the `New document` button.
+* New `editable_root` option that can be set to `false` to prevent editing of the editors root element.
+* New `editor.setEditableRoot` API that sets the editable state of the editor root element.
+* New `editor.hasEditableRoot` API that returns `true` or `false` depending on the editable state of the editor root element.
+* New `EditableRootStateChange` event that gets dispatched when the state of the editable root is changed.
+
+=== Improved
+* Screen readers are now able to announce the highlighted menu item of link comboboxes.
+* Now `icon` field for dialog footer `togglebutton`s is not mandatory.
+* Toolbar buttons and menu items were not disabled when they couldn't be used on noneditable content.
+* Updated toolbar "More" button tooltip text from "More..." to "Reveal or hide additional toolbar items".
```

Comment:
```suggestion
```

Reply:
This has been moved to `6.6` @brian-forte

Reply:
The suggested change and the original are identical. Consequently, I’m resolving this one.